python/feldera/sql_context.py

```diff
@@ -366,6 +380,86 @@ def foreach_chunk(self, view_name: str, callback: Callable[[pandas.DataFrame, in
         handler = CallbackRunner(self.client, self.pipeline_name, view_name, callback, queue)
         handler.start()
 
+    def connect_source_kafka(
+        self,
+        table_name: str,
+        connector_name: str,
+        config: dict,
+        fmt: JSONFormat | CSVFormat
+    ):
+        """
+        Tells Feldera to read the data from the specified kafka topic.
+
+        :param table_name: The name of the table.
+        :param connector_name: The unique name for this connector.
+        :param config: The configuration for the kafka connector.
+        :param fmt: The format of the data in the kafka topic.
```

Comment:
can you provide a reference to the format descriptions?

Reply:
Format takes an instance of  `JSONFormat` or `CSVFormat`.